templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
+        <div class="h1"> 
+		  <h1>TrackCat</h1>
+        </div>
+        <div class="h2">
```

Comment:
Similar comment as above - `<h2>` is a [built-in HTML tag](http://www.w3schools.com/tags/tag_hn.asp), so there's no need for this extra `<div>`. Just use `<h2>` instead of `<p>` to wrap the text, and style like like mentioned above. For more info, see